scripts/policygen/sources/jira/template.yml

```diff
@@ -0,0 +1,42 @@
+source:
+  host: {{ config.jira_domain }}
+  auth: basic_auth:jira
+  rules:
+    - route: "GET /rest/api/3/search"
```

Comment:
Do we need to include Jira Data Center as well? DC uses the v2 API i believe

Or would that be a different template

Reply:
I'll confirm with Bitgo. Hoping they're on Jira cloud. Are there any significant differences besides the path?